README.md

```diff
@@ -72,6 +72,29 @@ Aggregate is built using Gradle and Gretty, but we strongly recommend you use [I
 
 1. You should now be able to browse [http://localhost:8080](http://localhost:8080)
 
+### Deploy to AppEngine
+
+1. Follow the first 11 points of the official instructions for [Installing on AppEngine (Cloud)](https://opendatakit.org/use/aggregate/#Installing_an_App_Engine_Instance)
```

Comment:
Let's link to the new docs site at http://docs.opendatakit.org/aggregate-install/#installing-on-app-engine. 

Instead of numbering the steps, which can change and don't exist on the docs site, I would say, "Stop after Google configures the server, and before the tutorial."

Reply:
Done. After finishing your review I'll squash the unnecessary commits